src/server/services/wmo.ts

```diff
@@ -656,7 +657,13 @@ function formatWmoStatusLineItems(
           isVisible: statusItem.isVisible
             ? statusItem?.isVisible(index, wmoItem, today)
             : true,
-          documents: [], // NOTE: To be implemented
+          documents:
+            statusItem.status === 'Besluit'
+              ? wmoItem.documents?.map((doc) => ({
+                  ...doc,
+                  url: relayApiUrl(doc.url),
+                }))
```

Comment:
Een nieuw object bij elke iteratie maken is niet nodig. In dergelijke gevallen kun je beter `Object.assign(...)` gebruiken. Hiermee muteer je het bestaande object.
```suggestion
              ? wmoItem.documents?.map((doc) => Object.assign(doc, { url: relayApiUrl(doc.url) }))
```